### ISSUE TITLE
detailed typography for numbers

### ISSUE THREAD

**Issue by dylan-k** [dylan-k/tufte-bootstrap#8]
style for [oldstyle, tabular numbers etc](https://practicaltypography.com/alternate-figures.html). where appropriate.

**Comment by dylan-k** [comment]
lots of typography enhancements available, that may or may not be part of Bootstrap, reviewed at https://sparanoid.com/lab/opentype-features/

**Comment by seezee** [comment]
[Here's why I'm not using Gill Sans](https://www.typotheque.com/articles/re-evaluation_of_gill_sans) (Tufte's choice) in my implementation.

For my body text I chose [Merriweather](https://ebensorkin.wordpress.com/author/ebensorkin/) in part because it supports slashed zeros and has both lining and oldstyle figures, along with some other nice OpenType features.

You can use a utility class like

`.num {`
 ` font-feature-settings:'onum', 'pnum';`
 ` font-variant-numeric:oldstyle-nums proportional-nums;`
`}
`
`.lining {`
`  font-feature-settings:'lnum', 'tnum';`
`  font-variant-numeric:lining-nums tabular-nums slashed-zero;`
`}`

`.ord {`
`  font-variant-numeric:ordinal;`
`}`

to enable the features.

**Comment by dylan-k** [comment]
Do you use Merriweather Sans in your implementation, when sans type is needed?

**Comment by seezee** [comment]
I do, but only because I was already using Merriweather & it's a nice match — really just the path of least resistance. There are plenty of great sans typefaces out there, but Gill isn't one of them.

Gill does evoke a certain nostalgia whenever I see it and it's impossible for me to disassociate it from its history, but it just isn't legible enough to justify its use in any of my projects. The lowercase "L," uppercase "I," and numeral "1" are all too similar in appearance.

**Comment by dylan-k** [comment]
Agreed. For any kind of technical writing, or simply to avoid confusion, those letterform similarities are problematic. 

For this project, I think a large set of glyphs and the more detailed OpenType features. How do the Merriweather faces perform in that regard?

**Comment by seezee** [comment]
Both versions — serif and sans — come in 2 weights & styles (text + bold, normal + italic) & have various alternate ligatures & other OT features. The serif style definitely supports slashed zeros + oldstyle and lining figures. The lining figures are also tabular, whereas the oldstyle figures are proportional.

They both have lots of glyphs — so many that you may want to subset if you primarily write in only one language.

You can see them [in my Tufte adaptation here](https://mercury.photo/tufte-css-adaptation/). I've attached my `.SCSS` for you to look at; it may help. Just delete the `.txt` file extension and Bob's your uncle.

[style.max.scss.txt](https://github.com/dylan-k/tufte-bootstrap/files/3212745/style.max.scss.txt)

 
I'm using [Zach Leatherman's Critical FOFT with Data URI](https://www.zachleat.com/web/comprehensive-webfonts/) to speed page loads since my site is web-font heavy. If you look at my page source you can see that in action too. Spent the last couple of days writing code to automate implementing it but I've got a lot of work left on that.

Good luck!
